contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -88,8 +88,9 @@ def find_context_menu_item(self, text: str, raise_on_failure=True):
         behavior of context menus: They are not parented underneath the current main window, but are treated
         as a separate window.
         """
-        # Add an extra sleep. Context menu needs to show up before we search for it.
-        time.sleep(1)
+        wait_for_condition(lambda: self.find_control(
```

Comment:
Is there a reason that you are not adding a `max_time`?

Reply:
The default is five seconds and that feels about right: The one second we waited before was almost always enough and the video in the bug looks also as if five seconds is fine.